wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -500,7 +500,7 @@ pub async fn handle_wallet_command(
                 .ok_or(WalletCliError::NoWallet)?
                 .get_utxos(utxo_type.to_wallet_types())
                 .map_err(WalletCliError::Controller)?;
-            Ok(ConsoleCommand::Print(format!("{utxos:?}")))
+            Ok(ConsoleCommand::Print(format!("{utxos:#?}")))
```

Comment:
This is a very welcome change. I just stumbled upon that yesterday where I wished the output was pretty printed ^^